domains/auth/Auth.API/Tests/TestCryptographics.cs

```diff
@@ -0,0 +1,54 @@
+using API.Services;
+using API.Models;
+using Tests.Resources;
+using Xunit;
+using Xunit.Categories;
+using Moq;
+using System;
+
+namespace Tests;
+
+[UnitTest]
+public sealed class TestCryptographics
+{
+    private readonly Mock<ICryptographyService> _cryptographyService;
+
+    public TestCryptographics()
+    {
+        _cryptographyService = new Mock<ICryptographyService>();
```

Comment:
Do you create a mock?

Why? are we not specifically interested in testing your implementation,

I would write a test like:

```c#
var cryptoserver = new CryptographyService()

var data = "some random text"
var encrypted = cryptoserver.Encrypt(data);
var decrypted = cryptoserver.Decrypt(encrypted);

Assert.NotEqual(data, encrypted);
Assert.Equal(data, decrypted);
```